docs/user_guide/assessment/index.rst

```diff
@@ -11,6 +11,13 @@ fairness metrics, such as demographic parity and equalized odds.
 We will show how :class:`MetricFrame` can be used to evaluate the metrics
 identified during the course of a fairness assessment.
 
+Fairlean provides two primary ways of assessing fairness: :class:`MetricFrame`,
+which can be used to perform disaggregated analysis of a particular performance 
+metric (such as accuracy, false positive rate, etc.) across sensitive 
+groups, and a set of predefined fairness metrics that output a single 
+number. In the :ref:`perform_fairness_assessment`, we will dive further into
```

Comment:
Maybe "an aggregate number"?

Reply:
Changed from "a single number" to "an aggregate number"